frontend/src/components/SortServices.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react'
+import { Badge, IconButton, Menu, MenuItem } from '@material-ui/core'
+import { Icon } from './Icon'
+import { useDispatch } from 'react-redux'
+import { Dispatch } from '../store'
+import { makeStyles } from '@material-ui/core/styles'
+import { fontSize } from '@remote.it/components/lib/styles/variables'
+
+export interface ISortService {
+  value: number
+  name: string
+  sortService: (a: IService, b: IService) => number
+}
+const options: ISortService[] = [
+  {
+    value: 0,
+    name: 'Alpha A-Z',
+    sortService: (a: IService, b: IService) =>
+      a.name.toLowerCase() > b.name.toLowerCase() ? 1 : a.name.toLowerCase() < b.name.toLowerCase() ? -1 : 0,
+  },
+  {
+    value: 1,
+    name: 'Alpha Z-A',
+    sortService: (a: IService, b: IService) =>
+      a.name.toLowerCase() < b.name.toLowerCase() ? 1 : a.name.toLowerCase() > b.name.toLowerCase() ? -1 : 0,
+  },
+  { value: 2, name: 'Newest first', sortService: (a: IService, b: IService) => (a.createdAt > b.createdAt ? 1 : -1) },
+  { value: 3, name: 'Oldest first', sortService: (a: IService, b: IService) => (a.createdAt < b.createdAt ? 1 : -1) },
+]
+
+export const SortServices: React.FC = () => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
+  const { devices } = useDispatch<Dispatch>()
+  const open = Boolean(anchorEl)
+  const css = useStyles()
+  const [icon, setIcon] = React.useState('sort-amount-down')
+  const [dot, setDot] = React.useState(false)
+  const [optionSelected, setOptionSelected] = React.useState<ISortService>()
```

Comment:
We don't want local state, this should be removed, and only use the device model global state so that the setting is recalled after page changes.
```suggestion
```